### PR TITLE
Revert "Update 2023-kubecon-eu.md"

### PR DESCRIPTION
**Pull request by mkorbi**
Reverts cncf/tag-env-sustainability#76

In the preview the dates for Tuesday is wrong it must be the 18th and talks are double-listed on Wednesday

### CODE REVIEW

File: website/content/events/2023-kubecon-eu.md [hunk 1]
```diff
@@ -21,23 +21,15 @@ In the talk we will talk about the latest from the CNCF Environmental Sustainabi
 ### Sustainability related talks at a glance
 
 **Tuesday 19th of April**
```
Comment (mkorbi) [inline website/content/events/2023-kubecon-eu.md:23]: must be 18th

File: website/content/events/2023-kubecon-eu.md [hunk 2]
```diff
@@ -21,23 +21,15 @@ In the talk we will talk about the latest from the CNCF Environmental Sustainabi
...
-* [Power-Aware Scheduling](https://sched.co/1HyWC)
-* [GreenCourier](https://sched.co/1HyXM)
-* [TAG Project Meeting](https://sched.co/1JWOX) ⬅️ brought to you by the TAG ENV
+* Power-Aware Scheduling: https://sched.co/1HyWC
+* GreenCourier: https://sched.co/1HyXM
+* TAG Project Meeting: https://sched.co/1JWOX ⬅️ brought to you by the TAG ENV
 
 **Wednesday 19th of April**
-* [Accelerate Sustainable Community Collaboration](https://sched.co/1HyPf)
-* [Be the Change Our Planet Seeks: How YOU Can Contribute to Running Environment-Friendly Workloads on Kubernetes](https://sched.co/1HyW9)
```
Comment (mkorbi) [inline website/content/events/2023-kubecon-eu.md:30]: duplicate

File: website/content/events/2023-kubecon-eu.md [hunk 3]
```diff
@@ -21,23 +21,15 @@ In the talk we will talk about the latest from the CNCF Environmental Sustainabi
...
-* [Building a Sustainable, Carbon-Aware Cloud: Scale Workloads and Reduce Emissions](https://sched.co/1HyPo)
-* [Accelerate Sustainable Computing with Community Collaboration](https://sched.co/1HyPf)
```
Comment (mkorbi) [inline website/content/events/2023-kubecon-eu.md:32]: duplicate

**Review by mkorbi — COMMENTED** [review]
Please fix this

File: website/content/events/2023-kubecon-eu.md (hunk 2 above)
Comment (catblade) [reply website/content/events/2023-kubecon-eu.md:30]: Be the change is not a duplicate?

**Pull request closed without merging**